packages/next/build/webpack-config.ts

```diff
@@ -1460,6 +1461,9 @@ export default async function getBaseWebpackConfig(
       // replacement is done before its process.env.* handling
       (!isServer || webServerRuntime) &&
         new MiddlewarePlugin({ dev, webServerRuntime }),
+      process.env.ENABLE_FILE_SYSTEM_API === '1' &&
+        (!isServer || webServerRuntime) &&
+        new FunctionsManifestPlugin({ dev, webServerRuntime }),
```

Comment:
Will this plugin be enabled for both the server-web compiler and the client compiler?